rest/indextest/resync_test.go

```diff
@@ -58,7 +58,11 @@ func TestResyncWithoutIndexes(t *testing.T) {
 		// the sync docs index)
```

Comment:
NIT: remove this comment given the new configuration can have syncDocs and roles index?